bin/template/Dockerfile

```diff
@@ -11,12 +11,12 @@
 FROM node:slim
 
 # Labels for GitHub to read your action
-LABEL "com.github.actions.name"="Your action name"
-LABEL "com.github.actions.description"="A description of your action"
+LABEL "com.github.actions.name"="{{name}}"
+LABEL "com.github.actions.description"="{{{description}}}"
```

Comment:
Three `{{{` will tell Mustache not to escape the string. If a person wanted a description to include the word "it's", for example, this would render it as `it's` instead of `it&#39;s`. 🕺